SwedbankPaySDKTests/Utils/Assertions.swift

```diff
@@ -0,0 +1,21 @@
+import XCTest
+
+extension Result {
+    func assertSuccess(whereValueSatisfies assertions: (Success) -> Void = { _ in }) {
+        switch self {
+        case .success(let value):
+            assertions(value)
+        case .failure:
+            XCTFail("Result was \(self) (expected success)")
+        }
+    }
+    
```

Comment:
Codacy found an issue: [Lines should not have trailing whitespace.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6260585)